### PR TITLE
docs: add mlr3 post

### PR DESCRIPTION
**Pull request by m-muecke**
# Seal of Approval Checklist

-   [x] I have copied the [article template folder](https://github.com/rdatatable-community/The-Raft/tree/main/posts/YYYY-MM-DD-seal_of_approval-pkg_name), renamed it appropriately, and filled in all sections. See [this article](https://github.com/rdatatable-community/The-Raft/tree/main/posts/2024-05-30-seal_of_approval-dtplyr) for an example.

-   [ ] I am the maintainer this package.

OR

-   [ ] I have emailed the maintainer and cc'ed `rdatatable@gmail.com`, making them aware that I am submitting this article.

## Categories:

The relationship between this package and `data.table` is best described as:

-   [ ] **An extension package:** Adds to the internal functionality of `data.table`

-   [x] **An application package:** Uses `data.table` to accomplish a particular task or analysis.

-   [ ] **A bridge package:** Translates `data.table` syntax to different syntax or provides helper functions for transitioning between `data.table` and another object type.

-   [ ] **A partner package:** Not necessarily directly connected to `data.table`, but deliberately follows the [core philosophies of `data.table`](https://github.com/Rdatatable/data.table/issues/5693)

## Message for maintainers

If your package is accepted by The Raft, it will be passed along to the `data.table` developer team for final approval.

Would you like to pass along any information or messages directly to the team along with your blog post?

## Dev-do not edit

Review request @kellybodwin

### CODE REVIEW

File: posts/2024-09-24-seal_of_approval-mlr3/index.qmd [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+learners = c(glrn_rf_tuned, glrn_stack)
+bmr = benchmark(benchmark_grid(tasks, learners, rsmp("cv", folds = 3)))
+
+bmr$aggregate(msr("classif.acc"))
+```
+
+In this (much more complex!) example we chose two tasks and two learners and used automated
+tuning to optimize the number of trees in the random forest learner, and a machine learning
+pipeline that imputes missing data, collapses factor levels, and stacks models.
+We also showed basic features like loading learnersand choosing resampling strategies for benchmarking.
```
Comment (Anirban166) [inline posts/2024-09-24-seal_of_approval-mlr3/index.qmd]: learners and

Comment (Anirban166) [reply posts/2024-09-24-seal_of_approval-mlr3/index.qmd]: Also the para above could do with slight changes or subtle rephrasing, for e.g.:

> In this (much more complex!) example we chose two tasks and two learners and ...

In this (much more complex!) example, we chose two tasks and two learners.  *Insert new setence to go with the verb 'used' for both 'automated tuning' and the 'machine learning pipeline'*

or if you prefer to keep it as one sentence, then maybe we can insert a verb before 'a machine learning pipeline' (similar to 'chose' and 'used'), for e.g.:

In this more complex example, we selected two tasks and two learners, used automated tuning to optimize the number of trees in the random forest learner, and employed a machine learning pipeline that imputes missing data, consolidates factor levels, and stacks models.

**Review by Anirban166 — APPROVED** [review]
LGTM!

File: posts/2024-09-24-seal_of_approval-mlr3/index.qmd (hunk 1 above)
Comment (m-muecke) [reply posts/2024-09-24-seal_of_approval-mlr3/index.qmd]: The text is copied from the book, but you're right the sentence does drag a bit.

**Review by Anirban166 — APPROVED** [review]
Perfect. Thank you for the fix!

**Pull request merged**